<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsersManager</class>
 <widget class="QMainWindow" name="UsersManager">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>485</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>485</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Administrador de Usuarios</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="TopFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTableWidget" name="UsersList">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <column>
          <property name="text">
           <string>Lista de usuarios</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="DetailsFrame">
         <property name="title">
          <string>Detalles</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QFrame" name="DetailsTopFrame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QFormLayout" name="formLayout">
             <property name="fieldGrowthPolicy">
              <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
             </property>
             <property name="verticalSpacing">
              <number>6</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="IDLabel">
               <property name="text">
                <string>ID:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="IDInput">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="PasswordLabel">
               <property name="text">
                <string>Contraseña:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="ChangePassButton">
               <property name="text">
                <string>Cambiar</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="NewPassLabel">
               <property name="text">
                <string>Nueva contraseña:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="NewPassInput">
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="RepeatPassLabel">
               <property name="text">
                <string>Repetir contraseña:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="RepeatPassInput">
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QCheckBox" name="ShowPassInput">
               <property name="text">
                <string>Mostrar contraseña</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="CancelPassChangeButton">
            <property name="text">
             <string>Cancelar Cambio de Contraseña</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="PermissionsLabel">
            <property name="text">
             <string>Permisos</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QScrollArea" name="PermissionsArea">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>202</width>
               <height>320</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QGroupBox" name="LogReportFrame">
                <property name="title">
                 <string>Log de eventos</string>
                </property>
                <layout class="QFormLayout" name="formLayout_4">
                 <property name="fieldGrowthPolicy">
                  <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                 </property>
                 <item row="0" column="0">
                  <widget class="QLabel" name="CanReadAllLogLabel">
                   <property name="text">
                    <string>¿Puede ver todo?</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="CanDeleteLogLabel">
                   <property name="text">
                    <string>¿Puede eliminar?</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QCheckBox" name="CanReadAllLogInput">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QCheckBox" name="CanDeleteLogInput">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="UsersManagerFrame">
                <property name="title">
                 <string>Administrador de usuarios</string>
                </property>
                <layout class="QFormLayout" name="formLayout_2">
                 <item row="0" column="0">
                  <widget class="QLabel" name="CanReadUsersLabel">
                   <property name="text">
                    <string>¿Puede ver?</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QCheckBox" name="CanReadUsersInput">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="CanEditUsersLabel">
                   <property name="text">
                    <string>¿Puede modificar?</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <widget class="QLabel" name="CanDeleteUsersLabel">
                   <property name="text">
                    <string>¿Puede eliminar?</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QCheckBox" name="CanEditUsersInput">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="1">
                  <widget class="QCheckBox" name="CanDeleteUsersInput">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="CanCreateUsersLabel">
                   <property name="text">
                    <string>¿Puede crear?</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QCheckBox" name="CanCreateUsersInput">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="DriversManagerFrame">
                <property name="title">
                 <string>Administrador de transportistas</string>
                </property>
                <layout class="QFormLayout" name="formLayout_3">
                 <item row="0" column="0">
                  <widget class="QLabel" name="CanReadDriversLabel">
                   <property name="text">
                    <string>¿Puede ver?</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="CanCreateDriversLabel">
                   <property name="text">
                    <string>¿Puede crear?</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="CanEditDriversLabel">
                   <property name="text">
                    <string>¿Puede modificar?</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <widget class="QLabel" name="CanSuspendDriversLabel">
                   <property name="text">
                    <string>¿Puede suspender?</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="0">
                  <widget class="QLabel" name="CanDeleteDriversLabel">
                   <property name="text">
                    <string>¿Puede eliminar?</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QCheckBox" name="CanReadDriversInput">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QCheckBox" name="CanCreateDriversInput">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QCheckBox" name="CanEditDriversInput">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="1">
                  <widget class="QCheckBox" name="CanSuspendDriversInput">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="1">
                  <widget class="QCheckBox" name="CanDeleteDriversInput">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="ActionsFrame">
      <property name="title">
       <string>Acciones</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="NewButton">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>Nuevo</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/generic_button.png</normaloff>:/icons/generic_button.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ModButton">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>Modificar</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/generic_button.png</normaloff>:/icons/generic_button.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="SaveRegButton">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>Guardar</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/generic_button.png</normaloff>:/icons/generic_button.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="CancelModButton">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>Cancelar</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/generic_button.png</normaloff>:/icons/generic_button.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="DelButton">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>Eliminar</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/generic_button.png</normaloff>:/icons/generic_button.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
